Additional dependencies, which unfortunately is not enough to get
parallel builds to work.  Tested with -j12 which breaks :-(

Esben Haabendal <eha@doredevelopment.dk>

--- openssl-1.0.0c/Makefile.orig	2011-02-02 13:43:18.626143389 +0100
+++ openssl-1.0.0c/Makefile	2011-02-02 13:44:21.129322066 +0100
@@ -245,15 +245,15 @@
 
 build_crypto:
 	@dir=crypto; target=all; $(BUILD_ONE_CMD)
-build_ssl:
+build_ssl: build_crypto
 	@dir=ssl; target=all; $(BUILD_ONE_CMD)
-build_engines:
+build_engines: build_crypto
 	@dir=engines; target=all; $(BUILD_ONE_CMD)
-build_apps:
+build_apps: build_libs
 	@dir=apps; target=all; $(BUILD_ONE_CMD)
-build_tests:
+build_tests: build_libs
 	@dir=test; target=all; $(BUILD_ONE_CMD)
-build_tools:
+build_tools: build_libs
 	@dir=tools; target=all; $(BUILD_ONE_CMD)
 
 all_testapps: build_libs build_testapps
